app/models/user.rb

```diff
@@ -3,9 +3,6 @@ class User < ApplicationRecord
   has_many :likes, foreign_key: :author_id
   has_many :comments, foreign_key: :author_id
 
-  validates :name, presence: true
-  validates :post_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
```

Comment:
- [ ] Kindly make sure that the `User` model has validations, for name and post_counter, your validations got removed when you revert the commit